TenantFile/Api/appsettings.json

```diff
@@ -1,12 +1,18 @@
 {
+  "AllowedHosts": "*",
+  "CloudSQL": {
+    "ConnectionString": "Host=127.0.0.1;Uid=DATABASE_USER;Pwd=PASSWORD;Database=DATABASE_NAME"
+  },
+  "ConnectionStrings": {
+    "PostgresDBConection": "Host=localhost;Port=5432;Database=tenant_file;Username=postgres;Password=TenantFile"

```

Comment:
This looks like a personal DB, so let's not check it in.

Reply:
Removed local db